webpack/assets/javascripts/react_app/components/PasswordStrength/PasswordStrength.js

```diff
@@ -1,11 +1,16 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import ReactPasswordStrength from 'react-password-strength';
-
+import Loadable from 'react-loadable';
 import CommonForm from '../common/forms/CommonForm';
 
 import './PasswordStrength.scss';
 
+const LoadingComponent = () => <div> Loading... </div>;
```

Comment:
maybe use our Loader component? at least, the spinner from it (note text is also not extracted)